server/bin/pbench-index.py

```diff
@@ -454,6 +457,9 @@ def main(options, name):
 
     return res
 
+def indexing_failure(tb, error, tb_res):
```

Comment:
This is going to be fleshed out similarly to `pbench-report-status`. We'll need to use a different index and a different document type (TBD), so we'll need to tweak it in order to be able to reuse it here.